pkg/manager/member/pvc_resizer.go

```diff
@@ -305,16 +450,81 @@ func (p *pvcResizer) patchPVCs(ns string, selector labels.Selector, pvcQuantityI
 			if err != nil {
 				return err
 			}
-			klog.V(2).Infof("PVC %s/%s storage request is updated from %s to %s", pvc.Namespace, pvc.Name, currentRequest.String(), quantityInSpec.String())
-		} else if quantityInSpec.Cmp(currentRequest) < 0 {
-			klog.Warningf("PVC %s/%s/ storage request cannot be shrunk (%s to %s), skipped", pvc.Namespace, pvc.Name, currentRequest.String(), quantityInSpec.String())
-		} else {
-			klog.V(4).Infof("PVC %s/%s storage request is already %s, skipped", pvc.Namespace, pvc.Name, quantityInSpec.String())
+
+			// update volume status, so that we can observbe that volume is resizing immediately
+			if ctx.sourceVolumeStatus != nil {
+				if status, exist := ctx.sourceVolumeStatus[volName]; exist {
+					if status.CurrentCapacity.Cmp(status.ResizedCapacity) == 0 {
+						status.ResizedCapacity = quantityInSpec
+						status.ResizedCount = 0
```

Comment:
Why reset the count here?

Reply:
Where is the status of TC updated?

Reply:
In ord to observe that volume is resizing immediately after expending storage.
Otherwise we need to wait for next reconciliation.

Reply:
> Where is the status of TC updated?

modify `ctx.sourceVolumeStatus` will modify TC

Reply:
But the `ResizedCapacity` should have been updated in `updateVolumeStatus` function?

Reply:
yes, now don't need to reset the count here.

Reply:
Done